.circleci/config.yml

```diff
@@ -123,7 +123,7 @@ jobs:
             export GOOGLE_APPLICATION_CREDENTIALS=$HOME/extra/appcredentials.json
             hack/go-generate.sh
             go mod download
-            gotestsum --junitfile /tmp/test-results/go-test-report.xml ./... -- -race ./...
+            gotestsum --junitfile /tmp/test-results/go-test-report.xml ./... -- ./...
```

Comment:
on the surface of it, i suppose the race detector is a desirable feature of the test runner, despite additional time?

Reply:
yes I just wanted to gauge the extra time needed by race and possibly run a the test with race in a separate workflow, not required by image builds.

Reply:
benefit from removing race is negligible. Putting it again.